VAMobile/src/screens/HealthScreen/Appointments/UpcomingAppointments/UpcomingAppointmentDetails.tsx

```diff
@@ -104,7 +105,9 @@ const UpcomingAppointmentDetails: FC<UpcomingAppointmentDetailsProps> = ({ route
       title: getTranslation(isCovidVaccine ? 'upcomingAppointments.covidVaccine' : AppointmentTypeToID[appointmentType], t),
       startTime: getEpochSecondsOfDate(startDateUtc),
       endTime: getEpochSecondsOfDate(endTime),
-      location: name || '',
+      location: isIOS() && lat && long ? name || '' : address ? address.street + ' ' + address.city + ', ' + address.state + ' ' + address.zipCode : name || '',
```

Comment:
```suggestion
      location: isIOS() && lat && long ? name || '' : address ? `${address.street} ${address.city}, ${address.state} ${address.zipCode}` : name || '',
```
Template literals make it easier to read